### DISCUSSION TITLE
Ataching a image taken from the camera on a compositor model (Export to PDF)

### DISCUSSION THREAD

**Discussion by Scintsz** [opengisch/QField#4305] [Q&A]
I'm tring to make a photographic report of the inspection I made on the field. I prepared in QGIS a model that shows 4 pictures taken with the help of Qfield. The app doesn't atach the images on the compositor model that I made, nor does the coordinates of the picture using the EXIF expression to get the values. If I export the project folder back to QGIS everithing is working and in there I can make the Report, but the problem is that I would like also to export to PDF by atlas on the Qfield app.
Does QField have a problem when interacting with external files on the compositor? The pictures are in the standard DCIM folder inside the project.
In the image attached it is possible to see that the red rectangle represent the fields that are not being displayed on Qfield, both the images and the individual coordinates.

The expression of the image items are the relative path of the image itself, and for the exif coordinates is:
"""
to_dms(exif("foto_01_full", 'Exif.GPSInfo.GPSLongitude'), 'x', 3, 'aligned') ||
 ' e ' ||
to_dms(exif("foto_01_full", 'Exif.GPSInfo.GPSLatitude'), 'y', 3, 'aligned')
"""

In short, on QGIS the export works, but not in Qfield itself. The pictures are showing correctly in the attribute table.

![Report](https://github.com/opengisch/QField/assets/83991183/a8ce0c2b-db3b-4801-8498-0b1939b9bea7)
